content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md

```diff
@@ -11,104 +11,233 @@ keywords:
 - data backup and recovery
 ---
 
-Google Photos works best in TrueNAS using a Google Photos API key and [rclone](https://rclone.org/) token.
+Configuring a Google Photos cloud sync task requires a Google Photos API key and [rclone](https://rclone.org/) token.
+See the [rclone Google Photos backend documentation](https://rclone.org/googlephotos/) for more on using rclone to sync Google Photos, including features and limitations of the Google Photos API.
+
+This is a multi-part procedure that includes [generating Google API credentials](#creating-the-api-credentials), [installing and configuring rclone](#configuring-rclone) on your client OS, [creating cloud credentials](#creating-google-photos-cloud-credentials) for Google Photos on TrueNAS SCALE, and then [configuring the cloud sync task](#creating-the-cloud-sync-task).
```

Comment:
Okay, so this first sentence had me asking "How do I get this rclone token?" Clicking the link did not answer that question but I did find out more about it in the second link. I did a Google search, found an rclone forum post about the token that it is automatically generated through the connection or something like that.  
Line 15, why is it important to mention "features and limitation of the Google Photos API" here? How is this statement relevant to this particular procedure?
Line 17, the link provides the information on the token, called a remote, but I'm not sure if the "this is a multi-part procedure" is talking about what is in the link or the procedure in this article.
Might want to rewrite/organize theses introductory sentences for better information flow and more of an explanation about this.  

Reading through this article I think it could benefit from a summary overview procedure that steps them through each step, in the order they should complete them, with links to the sections. 
Some of the headers read like "before you begin" preparation steps, but if that is not the case then it can help the reader to know they should do this a particular way. For example:
1. Create your datasets on TruNAS. (with the link to the section about this.)
2. Create your rclone token (link to that section)
3. Add Google Photo authentication (link to that section)
etc. The parts on the direction and files is important but they might gloss over it if they don't know why that is important. It appears to be part of the advanced preparation or before you begin, or could just be really important about how Google Photos works. All this content is really excellent but I think it just needs that extra layer on top of it to direct the reader to what to do, the order to do it and the section with that content detail.